database/dbHelpers.js

```diff
@@ -0,0 +1,16 @@
+const db = require('./configKnex')
+
+module.exports = {
+  insert: user => {
+    return db('users').insert(user)
+  }, 
+  findUsername: user => {
+    return db('users').where({username: user}).first();
+  },
+  findUserDepartment: department => {
```

Comment:
Cleanest part of the whole code. (Could also change the `where` on the line below to be `where({department})`)